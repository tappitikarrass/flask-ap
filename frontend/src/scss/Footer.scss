@use "base16" as *;

footer {
    background-color: $base00;
    border-top: 1px $base02 solid;
    color: $base04;
    font-size: 0.8em;
    height: 100%;
    padding: 1.5em;
    text-align: center;
}
