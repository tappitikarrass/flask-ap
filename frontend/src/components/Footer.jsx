import React from 'react';
import '../scss/Footer.scss';

function Footer() {
  return (
    <footer>
      Â© 2022 Andrii Lytvyn
    </footer>
  );
}

export default Footer;
