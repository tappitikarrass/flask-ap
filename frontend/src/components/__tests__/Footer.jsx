import React from "react";
import { HashRouter } from "react-router-dom";
import { create, act } from "react-test-renderer";
import Footer from '../Footer';

describe('Footer', () => {
  it('',() => {
    let component;
    act(() => {
      component = create(
        <HashRouter>
          <Footer />
        </HashRouter>
      );
    });
    expect(component.toJSON()).toMatchSnapshot();
  });
});
