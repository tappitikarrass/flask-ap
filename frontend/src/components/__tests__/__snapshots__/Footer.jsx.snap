// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer  1`] = `
<footer>
  Â© 2022 Andrii Lytvyn
</footer>
`;
